app/component/map/tile-layer/DynamicParkingLots.js

```diff
@@ -98,7 +122,11 @@ class DynamicParkingLots {
       );
     }
 
-    const icon = DynamicParkingLots.getIcon(properties.lot_type);
+    const tags = properties.tags.split(',').reduce((newTags, tag) => {
+      return { ...newTags, [tag.split(':')[0]]: tag.split(':')[1] };
+    }, {});
```

Comment:
If backend returns tags as JSON-encoded string, we can remove this here.

Reply:
Sorry, I have bad news: https://github.com/stadtnavi/digitransit-ui/issues/604#issuecomment-863854611

Reply:
OK, then I will make sure the application won't break if tags does not contain key value pairs.

Reply:
Works as desired. I think we can merge.